TunnelVisionLabs.ReferenceAssemblyAnnotator/NullableAttributes.vb

```diff
@@ -94,7 +94,7 @@ Namespace Global.System.Diagnostics.CodeAnalysis
 
     ''' <summary>Specifies that the method will not return if the associated Boolean parameter is passed the specified value.</summary>
     <AttributeUsage(AttributeTargets.Parameter, Inherited:=False)>
-    Friend NotInheritable Class DoesNotReturnAttribute
+    Friend NotInheritable Class DoesNotReturnIfAttribute
```

Comment:
I believe it's an actual bug, as `DoesNotReturnAttribute` already appeared earlier in the file.

Reply:
Nice find. We have a test hole. @sharwell